Reneiski_Homework1/Reneiski_Homework1/Lecture5/Zavod.cs

```diff
@@ -0,0 +1,29 @@
+﻿using WorkerNamespace;
+
+namespace Reneiski_Homework1.Lecture5
+{
+    public class Zavod
```

Comment:
add 2 more methods:
that shows all workers (name+last name)
that shows all workers who has certain job position

Reply:
Fixed